migrations/00005-jobs.ts

```diff
@@ -0,0 +1,27 @@
+import { Sql } from 'postgres';
+
+export const up = async (sql: Sql<any>) => {
+  await sql`
+    CREATE TYPE job_status AS ENUM ('pending', 'queued', 'done', 'failed')
+  `;
+  await sql`CREATE TABLE jobs (
+    id                  SERIAL PRIMARY KEY,
```

Comment:
Heads up that `SERIAL` and `INT` types are only 32bits (compared to the 64bit `BIG` variants). Not sure if it would ever be a problem in prod, but using `bigint` and `bigserial` is basically free.

Reply:
Great point, I'll make the change